0001-basic-expiry-model.md

```diff
@@ -16,7 +16,8 @@ Values that are reasonable and that ensure that the protocol is secure for both
 
 ## Model
 
-The basic model we came up with is here: https://www.overleaf.com/read/sswbyytnfjhy (It was too difficult to express in markdown -- Talk to me for editable version).
+The basic model we came up with is here: https://www.overleaf.com/read/sswbyytnfjhy or here in [PDF format](assets/basic_expiry_model.pdf)(It was too difficult to express in markdown). 
```

Comment:
Massive nit-pick alert - Period goes inside the parenthesis if what is inside the parenthesis is a full sentence.
```suggestion
The basic model we came up with is here: https://www.overleaf.com/read/sswbyytnfjhy or here in [PDF format](assets/basic_expiry_model.pdf).  (It was too difficult to express in markdown.) 
```